README.md

```diff
@@ -14,42 +14,38 @@ Future work:
 ### Requirements
 
 * Requires Linux >= 4.5 (leverages the TCP Fast Open infrastructure).
-* Configure `$ sysctl -w net.ipv4.tcp_fastopen=5` to enable sending data in the
-opening SYN, regardless of cookie availability.
-* Check the [CI build script](.travis.yml) for a complete list of deps (based on Ubuntu Xenial).
+* Configure `$ sysctl -w net.ipv4.tcp_fastopen=5` to enable sending data in the opening SYN, regardless of cookie availability.
 
 ### Build & usage
 
-Build with CMake:
+Fetch the Git submodules:
 ```
 $ git submodule init && git submodule update
-$ mkdir build && cd build && cmake .. && make
 ```
 
-Usage (assuming a Transport Converter listening at 192.0.2.1:1234):
+The easiest way to build and run the tests is with the provided Dockerfile (which contains all deps):
 ```
-$ CONVERT_ADDR=192.0.2.1 CONVERT_PORT=1234 LD_PRELOAD=./libconvert_client.so curl https://www.tessares.net
+$ docker build -t tessares.net/libconvert .
+$ docker run --cap-add=NET_ADMIN --sysctl net.ipv4.tcp_fastopen=5 -v $PWD:/lc -t tessares.net/libconvert /bin/bash -c "mkdir /lc/build && cd /lc/build && cmake .. && make && make test"
```

Comment:
we could also add `--user "${UID}:${UID}"`

Reply:
Good idea, will change that tomorrow

Reply:
Well, tried that, but iptables & other utils are giving me troubles inside the container as they expect to be uid 0. Don't have much time to further work on this, will merge like this then